src/queue/workers.ts

```diff
@@ -19,12 +19,21 @@ export const expireGroup = async (groupId: string) => {
     }
 
     if (targetGroup.members && targetGroup.members.length > 0) {
-      const result = await User.updateMany(
+      // remove group id from all members who have joined the group
+      await User.updateMany(
         { _id: targetGroup.members },
         {
           currentGroup: undefined,
         }
       );
+
+      // remove group id from all invited members who have not joined the group
+      await User.updateMany(
+        { _id: targetGroup.invitedMembers },
+        {
+          $pull: { invitedGroups: new ObjectId(groupId) },
```

Comment:
I don't believe you need to construct a Object Id here, you can just pass mongoose the string